libs/ddd-angular/src/generators/feature/generator.ts

```diff
@@ -3,10 +3,13 @@ import { DddFeatureGeneratorSchema } from './schema';
 import dddFeatureGenerator from '@angular-architects/ddd/src/generators/feature';
 import { applicationGenerator } from '@nrwl/angular/generators';
 import { removeGenerator } from '@nrwl/workspace';
+import { guardValidDomain } from '../../helpers';
 
 const APP_NAME = 'bobby-john-boratos';
 
 export default async function (tree: Tree, options: DddFeatureGeneratorSchema) {
```

Comment:
Shouldn't the app configured in `dddFeatureGenerator` be given the name from options.name instead of bobbyJohnBoratos?

Reply:
Not there anymore :0